sources/api/apiserver/src/bin/apiserver.rs

```diff
@@ -131,7 +131,7 @@ async fn run() -> Result<()> {
     let args = parse_args(env::args());
 
     // TerminalMode::Mixed will send errors to stderr and anything less to stdout.
```

Comment:
This comment is no longer valid, similarly in other files where `TerminalMode::Mixed` was used.